### ISSUE TITLE
Spinning in auto

### ISSUE THREAD

**Issue by KenwoodFox** [FRC-1721/1721-ChargedUp#19]
@Kredcool 's issue, robot spins in auto and wont follow ramsette trajectories properly.

**Comment by Kredcool** [comment]
This is proof that I am not smart. The issue is the left side is getting less voltage the more it goes up
```
[29.56] (0.0, 0.0)
[29.58] (9.918212890625e-07, 1.0938644409179688e-06)
[29.60] (9.918212890625e-07, 1.0938644409179688e-06)
[29.62] (9.918212890625e-07, 1.0938644409179688e-06)
[29.64] (9.918212890625e-07, 1.0938644409179688e-06)
[29.66] (9.918212890625e-07, 1.0938644409179688e-06)
[29.68] (9.918212890625e-07, 1.0938644409179688e-06)
[29.70] (9.918212890625e-07, 1.0938644409179688e-06)
[29.72] (9.918212890625e-07, 1.0938644409179688e-06)
[29.74] (9.918212890625e-07, 1.0938644409179688e-06)
[29.76] (9.918212890625e-07, 1.0938644409179688e-06)
[29.78] (9.918212890625e-07, 1.0938644409179688e-06)
[29.80] (9.918212890625e-07, 1.0938644409179688e-06)
[29.82] (9.918212890625e-07, 1.0938644409179688e-06)
[29.84] (9.918212890625e-07, 1.0938644409179688e-06)
[29.86] (9.918212890625e-07, 1.0938644409179688e-06)
[29.88] (9.918212890625e-07, 1.0938644409179688e-06)
[29.90] (9.918212890625e-07, 1.0938644409179688e-06)
[29.92] (9.918212890625e-07, 1.0938644409179688e-06)
[29.94] (9.918212890625e-07, 1.0938644409179688e-06)
[29.96] (9.918212890625e-07, 1.0938644409179688e-06)
[29.98] (9.918212890625e-07, 1.0938644409179688e-06)
[30.00] (9.918212890625e-07, 1.0938644409179688e-06)
[30.02] (9.918212890625e-07, 1.0938644409179688e-06)
[30.04] (9.918212890625e-07, 1.0938644409179688e-06)
[30.06] (0.8311403995423345, 0.8311405015854864)
[30.08] (0.34437661472863407, 0.3443767167717859)
[30.10] (0.34848251397826085, 0.3484826160214127)
[30.12] (0.30811493225094183, 0.3881730519853679)
[30.14] (0.3113931308768076, 0.41426215478098793)
[30.16] (0.2874526732631854, 0.41871049899149576)
[30.18] (0.273510011328871, 0.43348521119691547)
[30.20] (0.27771896163270704, 0.4545025666671305)
[30.22] (0.2660012751469248, 0.4603975749004341)
[30.24] (0.2524663534509049, 0.4807679749786183)
[30.26] (0.2554644854747468, 0.5013748592219229)
[30.28] (0.24991138404032773, 0.5069360185319888)
[30.30] (0.24316608860877967, 0.5282825731913638)
[30.32] (0.24543566939444647, 0.554316166445097)
[30.34] (0.23915875816404789, 0.5590812998113016)
[30.36] (0.27846892460152206, 0.5526749665713356)
[30.38] (0.28176321130502135, 0.5553664068720436)
[30.40] (0.303789930864682, 0.5610907433069172)
[30.42] (0.30793514048243087, 0.5820842885213606)
[30.44] (0.31273626458487497, 0.5868818957382481)
[30.46] (0.3221044153742061, 0.5962473228245154)
[30.48] (0.3262997702639926, 0.6119676128972942)
[30.50] (0.3315532800373069, 0.6170348917125955)
[30.52] (0.3299551074503795, 0.6268426876575353)
[30.54] (0.3348339180341988, 0.6431228172745056)
[30.56] (0.3326641044406874, 0.6467507072066982)
[30.58] (0.3313918935213118, 0.6567924501086886)
[30.60] (0.3362907468959594, 0.6617764974362009)
[30.62] (0.3360428277465074, 0.6729513694167091)
[30.64] (0.3392220921914001, 0.6875558006995832)
[30.66] (0.3384857313657428, 0.6925285311181384)
[30.68] (0.3426839474906722, 0.6966008886429822)
[30.70] (0.3487156487826472, 0.7140255840384113)
[30.72] (0.3460472609619465, 0.7173011273706456)
[30.74] (0.3512873933335623, 0.728013801527436)
[30.76] (0.3553670386976207, 0.7436229490580155)
[30.78] (0.36167202398036413, 0.7497827506314457)
[30.80] (0.3647536947763435, 0.7586867729169345)
[30.82] (0.3694192466610101, 0.7690600455003591)
[30.84] (0.3751298963899704, 0.7749086207009058)
[30.86] (0.37449177987730375, 0.7911421781032764)
[30.88] (0.3779603608316255, 0.7948762097158854)
[30.90] (0.3839420899925624, 0.7982896315836377)
[30.92] (0.38846499766229126, 0.8184148131721924)
[30.94] (0.391846492080845, 0.8201481452063282)
[30.96] (0.39397160311138557, 0.8311418698476714)
[30.98] (0.39624433726476427, 0.8433979032619585)
[31.00] (0.4029227157159657, 0.8480199226771161)
[31.02] (0.40164402459058723, 0.8552082343458878)
[31.04] (0.40564466324007675, 0.8762885924934104)
[31.06] (0.4052281764460064, 0.8788386584824781)
[31.08] (0.41208959259528244, 0.8915418797215857)
[31.10] (0.41474684861481104, 0.8938319983552251)
[31.12] (0.42147548084001507, 0.9067452142312804)
[31.14] (0.42799039527077065, 0.920823792463207)
[31.16] (0.42472174160295906, 0.9300114268963134)
[31.18] (0.42453402188911493, 0.9377733409410947)
[31.20] (0.43293940071195347, 0.9482765260526211)
[31.22] (0.4379750831118472, 0.9550590442153938)
[31.24] (0.4398186503846465, 0.9648061129839083)
[31.26] (0.4481325551140604, 0.9845320379013248)
[31.28] (0.45167936023388644, 0.9876476541868442)
[31.30] (0.44952511587070043, 0.9972605494957042)
[31.32] (0.4596076879498392, 1.0135144139556873)
[31.34] (0.453993355384832, 1.0230068681289561)
[31.36] (0.4615516976357246, 1.0320633062383968)
[31.38] (0.46716390550530595, 1.0374345232637587)
[31.40] (0.47351908621623684, 1.049674418630648)
[31.42] (0.477510577157455, 1.064845249124009)
[31.44] (0.47710931456077205, 1.0703397785670963)
[31.46] (0.48158855751456764, 1.0802344439807883)
[31.48] (0.48089255908549255, 1.1023074879411525)
[31.50] (0.48930276058511013, 1.1053857900539636)
[31.52] (0.48934918981418213, 1.1165899674588058)
[31.54] (0.49523084884055313, 1.1278674297555478)
[31.56] (0.505555331651243, 1.1328847506132196)
[31.58] (0.506474446140143, 1.1452130800016023)
[31.60] (0.5141108812425722, 1.1530946031069065)
[31.62] (0.5130414828360871, 1.1686657988945193)
[31.64] (0.5178814372131156, 1.1791448001766058)
[31.66] (0.5180639809993862, 1.1849778614143052)
[31.68] (0.5324091844138008, 1.1995237041903655)
[31.70] (0.5356442979163785, 1.2144615253764528)
[31.72] (0.537061857087835, 1.2213347397844145)
[31.74] (0.5404285491381611, 1.2277772642443698)
[31.76] (0.5412495943322574, 1.2489859061741133)
[31.78] (0.5498294480854748, 1.2510782702783911)
[31.80] (0.5502410129600009, 1.2686118157322288)
[31.82] (0.5598792711985416, 1.2806953818773419)
[31.84] (0.5553072687563319, 1.2986091768900723)
[31.86] (0.5688182691457051, 1.2994548686582934)
[31.88] (0.563104898286584, 1.312321454006164)
[31.90] (0.5793065247679796, 1.3317855745413567)
[31.92] (0.5808101585133731, 1.3389928472450219)
[31.94] (0.5799415427823441, 1.3495178683398175)
[31.96] (0.5825989041042554, 1.3632490999749503)
[31.98] (0.5932820124096649, 1.3795414759417468)
[32.00] (0.5914569944172137, 1.3833594692383344)
[32.02] (0.596412111185397, 1.400001014889947)
[32.04] (0.6158202718323853, 1.4124634025652965)
[32.06] (0.5984016158497041, 1.4236218087464012)
[32.08] (0.6132031990898059, 1.4337801925707407)
[32.10] (0.6236097592769503, 1.4441467763132931)
[32.12] (0.6186241033404475, 1.4498203497551385)
[32.14] (0.629414253283102, 1.4726215471196156)
[32.16] (0.6354541527379379, 1.4786331676616076)
[32.18] (0.6356326691845104, 1.4902117525620273)
[32.20] (0.6463203590495181, 1.5061670335458874)
[32.22] (0.6454255754295865, 1.5112940138750872)
[32.24] (0.6502704408726554, 1.527540751472027)
[32.26] (0.6619324281028627, 1.5505837471493886)
[32.28] (0.6576033613477539, 1.5519533235888114)
[32.30] (0.6721057075245919, 1.571689041147959)
[32.32] (0.6735596757322674, 1.5849040148677165)
[32.34] (0.678832387000648, 1.595404065865332)
[32.36] (0.6865117289998017, 1.6143941552466579)
[32.38] (0.690027675263858, 1.6186911927951497)
[32.40] (0.7005341389825641, 1.6263032974926475)
[32.42] (0.686146338113458, 1.658758508345267)
[32.44] (0.70970448945133, 1.6601997620793174)
[32.46] (0.7248632912887895, 1.6734664571367368)
[32.48] (0.7084167026463912, 1.703000526115789)
[32.50] (0.7214089981468766, 1.7062752070117029)
[32.52] (0.7248564221040996, 1.721102813735707)
[32.54] (0.7422333931384805, 1.7279663502507616)
[32.56] (0.7266335953867809, 1.7580027340480122)
[32.58] (0.7394574282800328, 1.7640662829844347)
[32.60] (0.7514391148729755, 1.7754726079315342)
[32.62] (0.7538815689906261, 1.7954335135468198)
[32.64] (0.7697763729893589, 1.7984361088779424)
[32.66] (0.7550190438373745, 1.8331476912748692)
[32.68] (0.7658512007467994, 1.8413607390392404)
[32.70] (0.7758299932460493, 1.856450629811512)
[32.72] (0.7724129788088576, 1.8597134218916096)
[32.74] (0.7889187006618786, 1.8876647810372214)
[32.76] (0.791621861656602, 1.9002065281850276)
[32.78] (0.7995979863962226, 1.9141547508254901)
[32.80] (0.7966663269571271, 1.9288134217970312)
[32.82] (0.809870689254218, 1.9528062570284215)
[32.84] (0.816595989009435, 1.9605440496192204)
[32.86] (0.8203108060539914, 1.980259408797337)
[32.88] (0.8441260907825339, 1.972592048556697)
[32.90] (0.8274606025717683, 2.024185352914708)
[32.92] (0.8439338933612535, 2.026839821941491)
[32.94] (0.8514504033582129, 2.039987485584011)
[32.96] (0.8551191860549483, 2.0607344649700052)
[32.98] (0.8599790797916524, 2.076341638782112)
[33.00] (0.8849660433408639, 2.0706442741423325)
[33.02] (0.8526496731348177, 2.1209064983453096)
[33.04] (0.8836510946994725, 2.1335941731222423)
[33.06] (0.9050368026755717, 2.1179337512543395)
[33.08] (0.8729170241390188, 2.1722385204726913)
[33.10] (0.8945785738192877, 2.1620116219712235)
[33.12] (0.898051129763686, 2.1825542882215534)
[33.14] (0.9110161860579549, 2.2061619316922627)
[33.16] (0.9136876833014527, 2.214356308312511)
[33.18] (0.9177468451656678, 2.23534297778643)
[33.20] (0.9247625635326175, 2.2614580728907594)
[33.22] (0.939121387904786, 2.273096743023995)
[33.24] (0.9649805441936884, 2.265323410938428)
[33.26] (0.9312185305873582, 2.3247098215803144)
[33.28] (0.9612605186504866, 2.3185120031822075)
[33.30] (0.9866611025615284, 2.323392235564261)
[33.32] (0.9493385648702946, 2.39368909310223)
[33.34] (0.9788223380938545, 2.3816583017757984)
[33.36] (1.0145860691588835, 2.3830404270801187)
[33.38] (0.973184527001359, 2.433846083352428)
[33.40] (0.9990979083089191, 2.4347366872353575)
[33.42] (0.0, 0.0)
```

**Comment by KenwoodFox** [comment]
Looks like this got fixed in 86bf0576ab202423378027c13f2df17568e4c361
